Makefile.defs

```diff
@@ -480,8 +480,10 @@ else
 	#          &  https://github.com/llvm/llvm-project/blob/1c4b5a8dfed853ca8cbee98f274ff0a173186586/clang/lib/Frontend/CompilerInvocation.cpp#L2136-L2164
 	#       FWIW, the C++ move to gnu++14 by default dates back to Clang 6:
 	#       c.f., https://github.com/llvm-mirror/clang/commit/466d8da5f89b1a780f735c86f414fa69ce63221b
-	CSTD_FLAGS:=-std=gnu11
-	CXXSTD_FLAGS:=-std=gnu++14
+
+	# TBA: Do we need this? It seems to actually break clang/OSX in tesseract..
```

Comment:
@NiLuJe Is it ok to drop this and instead add case-by-case if really needed - ie things where autotools don't figure this out on their own? It explodes on clang due to tesseract and some mupdf dependencies mixing up CFLAGS vs CXXFLAGS.

Reply:
I suspect it's unnecessary on macOS, since even truly ancient versions of XCode should default to a recent-enough Clang that would make this redundant.

(IIRC, the main user of this hack was the old stock PB TC. WHich the current set of TCs, this is arguably a hairy NOP ;)).